src/lib/authentification.ts

```diff
@@ -162,6 +168,7 @@ export async function internetIdentityLogout() {
 	const autClient = await AuthClient.create();
 	await autClient.logout();
 }
+import { AccountIdentifier, type Account, SubAccount } from '@dfinity/ledger-icp';
```

Comment:
Let's move this import to the top for consistency.

Reply:
unused import my bad. Got rid of it